lib/pullrequest-decorator-reporter.js

```diff
@@ -128,16 +135,9 @@ class PullRequestDecoratorReporter {
     }
   }
 
-  done(error, summary) {
+ done(error, summary) {
     if (!this.options.export) {
-      return this.githubService.createPullRequestCheck(
-        this.buildReport(this.context.assertions, summary.collection.items),
-        this.options,
-      )
-        .catch((exception) => { console.error(`[+] An error occured : ${exception}`); })
-        .finally(() => {
-          console.log(`[+] Finished collection: ${this.options.collectionName}`);
-        });
+      return decoratePullRequest(this.githubService, this.buildReport(this.context.assertions, summary.collection.items), this.options);
```

Comment:
You can have decoratePullRequest as part of PullRequestDecoratorReporter like buildReport and buildItemsReport methods. 
And you won't need to pass this.githubService, as you can call this.githubService within decoratePullRequest

Reply:
Done